index.js

```diff
@@ -22,14 +21,22 @@ const exec = (cmd, args) => {
 	).filter(Boolean);
 };
 
+const install = (bin, useYarn) => () => execa(bin, ['install'])
```

Comment:
Aha. Makes sense. I'll create two issues for these features and we can continue the discussion. I can probably work on this tomorrow so it might be better to wait a little longer to finish this PR.